packages/templates/wasm/rust/package.json

```diff
@@ -8,8 +8,8 @@
     "build": "npx polywrap build",
     "deploy": "npx polywrap deploy",
     "test": "yarn test:e2e && yarn test:workflow",
-    "test:e2e": "yarn test:e2e:codegen && jest --passWithNoTests --runInBand --verbose",
-    "test:e2e:codegen": "npx polywrap codegen -m ./src/__tests__/types/polywrap.app.yaml -g ./src/__tests__/types/wrap",
+    "test:e2e": "yarn test:e2e:codegen && cargo test --release",
+    "test:e2e:codegen": "npx polywrap codegen -m ./tests/types/polywrap.app.yaml -g ./tests/types/wrap",
     "test:workflow": "npx polywrap test"
   },
   "devDependencies": {
```

Comment:
i think we can also remove `jest` and its relatives from here?

Reply:
Done!